assets/js/atomic/blocks/product-elements/button/style.scss

```diff
@@ -50,6 +61,17 @@
 		margin-left: auto;
 		white-space: normal;
 		word-break: break-word;
+		width: 150px;
+
+		span {
+
+			&.wc-block-scrollToUp {
+				animation: scrollToUp 0.1s linear 1 normal;
+			}
+			&.wc-block-scrollFromDown {
+				animation: scrollFromDown 0.1s linear 1 normal;
+			}
+		}
```

Comment:
This should probably be wrapped with [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).